skrf/util.py

```diff
@@ -117,6 +117,20 @@ def wrapper(func: Callable) -> Callable:
         return func
     return wrapper
 
+def figure(*args, **kwargs) -> Figure:
```

Comment:
shouldn't it in `plotting.py` instead than in `util.py` ?

Reply:
I think it is related to d832ef6 with matplotlib being optionnal. Because `suplots` were already there I added `figure`.
It is also true both concern plotting.

Reply:
Afterthought: the optional matplotlib dependency is pertinent for a method that shall work without matplotlib (e.g. if it is only an optional verbose output). Plotting functions cannot be performed without it, because their sole purpose is to plot.